assets/js/fquery.js

```diff
@@ -1,10 +1,8 @@
-export const all = (selector, event, listener) => {
-  document.querySelectorAll(selector).forEach(e => {
-    e.addEventListener(event, listener);
-  });
+export const forEach = (selector, event, listener) => {
```

Comment:
You didn't have any issue to debug as anonymous function doesn't display any name in the stack trace?